NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -72,6 +74,91 @@ public StandaloneQuery(StandaloneContext standaloneContext, IConfiguration confi
                 }
             );
 
+            Field<ListGraphType<RootStateDiffType>>(
+                name: "diffs",
```

Comment:
I personally don't like it to work implicitly, I think it's better to accept them all as Required instead of Nullable, but it's okay if they're not accepted. If you want to give a default, please write about the default behavior in the GraphQL description.

Reply:
I did my best to match the setup of EmptyChronicles.
I got rid of nulls in responses and made all arguments mandatory. Please check it